app/src/main/java/com/likeminds/feedsx/base/customview/LikeMindsToolbar.kt

```diff
@@ -0,0 +1,38 @@
+package com.likeminds.feedsx.base.customview
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.graphics.Color
+import android.util.AttributeSet
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.appcompat.widget.Toolbar
+import androidx.core.view.children
+import com.likeminds.feedsx.utils.branding.BrandingData
+
+
+class LikeMindsToolbar : Toolbar {
+    constructor(context: Context) : super(context) {
+        initiate(null)
+    }
+
+    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
+        initiate(attrs)
+    }
+
+    constructor(context: Context, attrs: AttributeSet?, defStyle: Int) : super(
+        context,
+        attrs,
+        defStyle
+    ) {
+        initiate(attrs)
+    }
+
+    private fun initiate(attrs: AttributeSet?) {
+        this.setBackgroundColor(if(BrandingData.isBrandingBasic) Color.WHITE else BrandingData.currentAdvanced!!.first)
```

Comment:
assign color to variable first and use it is inside `this.setBackgroundColor`

Reply:
Got it!